src/datasets/table.py

```diff
@@ -46,13 +46,37 @@ def _in_memory_arrow_table_from_buffer(buffer: pa.Buffer) -> pa.Table:
     return table
 
 
+def _memory_mapped_arrow_stream_from_file(filename: str):
```

Comment:
I think this function is not necessary, we can call pa.memory map directly

Reply:
Tip applied